### DISCUSSION TITLE
error: RPC failed; curl 56 Send failure: Connection was reset

### DISCUSSION THREAD

**Discussion by Mummina** [community/community#33407] [New to GitHub]
please help me out regarding this issue and I tried multiple ways but its not working.

$ git push -u -f origin --all

Enumerating objects: 315, done.
Counting objects: 100% (315/315), done.
Delta compression using up to 8 threads
Compressing objects: 100% (189/189), done.
Writing objects: 100% (315/315), 336.28 MiB | 217.12 MiB/s, done.
Total 315 (delta 120), reused 313 (delta 118), pack-reused 0
error: RPC failed; curl 56 Send failure: Connection was reset
send-pack: unexpected disconnect while reading sideband packet
fatal: the remote end hung up unexpectedly
Everything up-to-date

**Comment by airtower-luna** [comment]
The [documentation mentions](https://docs.github.com/en/repositories/working-with-files/managing-large-files/about-large-files-on-github#file-size-limits):

> GitHub blocks pushes that exceed 100 MB.

Your push is significantly larger than that:

>  Writing objects: 100% (315/315), 336.28 MiB | 217.12 MiB/s, done.

If you have a lot of small commits, you can push them in small parts of the repository history instead of all of them at once. If you have individual large files and commits you'll need to check the alternatives mentioned in the linked documentation article.
